Maven stable releases no longer work with maven-source-plugin 3.3.0+

### Checked for duplicates

Yes - I've already checked

### 🐛 Describe the bug

In several Java apps, I am encountering failures when trying to tag a stable release:
* https://github.com/NASA-PDS/registry-mgr-legacy/actions/runs/8640957511
* https://github.com/NASA-PDS/registry-harvest-legacy/actions/runs/8640966587
* https://github.com/NASA-PDS/search-ui-legacy/actions/runs/8640666910

Others I can't remember right now...

### 🕵️ Expected behavior

I expected the stable release to complete successfully

### 📜 To Reproduce

1. Trigger a stable release on another Java repo that uses the default stable publication steps.
2. Note this error in the GitHub action execution:
> `Caused by: org.apache.maven.plugin.MojoExecutionException: Presumably you have configured maven-source-plugn to execute twice times in your build. You have to configure a classifier for at least on of them.`

### 🖥 Environment Info

GitHub Action World

### 📚 Version of Software Used

stable

### 🩺 Test Data / Additional context

Tried numerous things to try to fix this, and may have come to a conclusion but not sure.
* At first I think this was because we define the `maven-source-plugin` in the [maven parent release profile](https://github.com/NASA-PDS/pdsen-maven-parent/blob/main/pom.xml#L264), so I removed any references from any of the child projects (even though oddly, some POMs had no reference to this 🤔 ) - **no joy** 😢 

* Next, I double-triple checked we were doing something like this in our parent POM: https://stackoverflow.com/questions/76305897/maven-build-fails-after-upgrading-to-maven-source-plugin-from-3-2-1-to-3-3-0 . Even [removed the phase from configuration ](https://github.com/NASA-PDS/pdsen-maven-parent/commit/862a0e8e146e9a251a81f1daa4c7f76ae971195b)in case that has anything to do with it. - **no joy** 😭 

* After some additional googling, it seems like this may be because we are calling "duplicate" lifecycle phases in our maven execution. our default is `['mvn', '--quiet', '--errors', '--activate-profiles', 'release', 'clean', 'package', 'site', 'deploy']`, and it appears `deploy` actually runs the `package` lifecycle phase. When I updated the maven-stable-artifact-phases for roundup, it worked as expected 🎉 

https://github.com/NASA-PDS/search-ui-legacy/blob/main/.github/workflows/stable-cicd.yaml#L76


### 🦄 Related requirements

_No response_

### ⚙️ Engineering Details

Let's update the default maven stable artifact build to run

--- a/src/pds/roundup/main.py
+++ b/src/pds/roundup/main.py
@@ -52,11 +52,11 @@ def _parseArgs():
     # Maven 😩
     group = parser.add_argument_group('Maven phases (or goals), comma-separated')
     group.add_argument('--maven-test-phases', help='🩺 Test (%(default)s)', default='test')
-    group.add_argument('--maven-doc-phases', help='📚 Documentation (%(default)s)', default='package,site,site:stage')
-    group.add_argument('--maven-build-phases', help='👷‍ Build (%(default)s)', default='compile')
+    group.add_argument('--maven-doc-phases', help='📚 Documentation (%(default)s)', default='clean,site,site:stage')
+    group.add_argument('--maven-build-phases', help='👷‍ Build (%(default)s)', default='install')
     group.add_argument(
         '--maven-stable-artifact-phases',
-        help='😌 Stable artifacts (%(default)s)', default='clean,package,site,deploy'
+        help='😌 Stable artifacts (%(default)s)', default='clean,site,site:stage,deploy'
     )
     group.add_argument(
         '--maven-unstable-artifact-phases',

